packages/babel-plugin-proposal-optional-chaining/test/fixtures/general/parenthesized-member-call/exec.js

```diff
@@ -0,0 +1,25 @@
+let count = 0;
```

Comment:
Consider rewrite the test into something similar with https://github.com/babel/babel/blob/785b6e8bde1419b3ea7d219cd8caf9dab043edfb/packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/optional-chain-private-call-with-transform/exec.js

for better coverage.